server/src/main/java/org/apache/seata/server/spring/web/RestControllerBeanPostProcessor.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.seata.server.spring.web;
+
+import org.apache.seata.core.rpc.netty.http.ControllerManager;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.RestController;
+
+@Component
+public class RestControllerBeanPostProcessor implements BeanPostProcessor {
+
+    @Override
+    public Object postProcessAfterInitialization(Object bean, String beanName) throws BeansException {
+        if (!bean.getClass().isAnnotationPresent(RestController.class)) {
+            return bean;
+        }
+        ControllerManager.addHttpController(bean);
```

Comment:
可以将解析逻辑放到这，或者spring-boot-starter的server模块去，避免core模块对spring的强依赖
You can put the parsing logic here, or the server module of spring-boot-starter to avoid the strong dependence of the core module on spring.